java/README.md

```diff
@@ -47,6 +47,25 @@ java -version
 
 _Note: Publishing to Maven is pending; the examples below will be available once artifacts are published. You can also download artifacts from the [GitHub release](https://github.com/valkey-io/valkey-glide/releases/tag/v1.0.0) and add the jar file to the project._
 
+You can publish the release to local maven by downloading the artefacts from the latest [release](https://github.com/valkey-io/valkey-glide/releases). You can update to local maven using the following example for osx-aarch64:
+
+```bash
+unzip java-aarch64-apple-darwin.zip
+jar xf bundle-java-aarch64-apple-darwin.jar
```

Comment:
I think first 2 steps are not needed anymore

Reply:
3rd line worked on my PC - I skipped those 
```
mvn install:install-file -Dfile=bundle-java-x86_64-unknown-linux-gnu.jar -DgroupId=io.valkey -DartifactId=valkey-glide -Dversion=1.0.0 -Dpackaging=jar -DPom=valkey-glide-1.0.0.pom
```

Reply:
I think you have to unzip

Reply:
pointing directly to the bundle is a good idea

Reply:
no zips uploaded to the release anymore